drivers/ethernet/Kconfig.stm32_hal

```diff
@@ -112,4 +112,46 @@ config ETH_STM32_MODE_HALFDUPLEX
 
 endif # !ETH_STM32_AUTO_NEGOTIATION_ENABLE
 
+if SOC_SERIES_STM32F7X || SOC_SERIES_STM32H7X
+
+config PTP_CLOCK_STM32_HAL
+	bool "STM32 HAL PTP clock driver support"
+	default y
+	depends on PTP_CLOCK || NET_L2_PTP
+	help
+	  Enable STM32 PTP clock support.
+
+if PTP_CLOCK_STM32_HAL
```

Comment:
Please remove the if statement here and use depends on in appropriate config menus.

Reply:
Updated, thanks!